uptane-standard.md

```diff
@@ -620,6 +622,8 @@ The Image repository MAY require authentication for read access.
 
 ### Director Repository {#director_repository}
 
+The first repository name listed in the repository mapping metadata {{repo_mapping_meta}} is to be known as the Director repository.
```

Comment:
Yeah, I'm not sure I like this convention, would break backwards-compatibility for existing implementations.

Reply:
This is the standard.  There was no policy previously, so I'm not sure what we'd be trying to be backwards-compatible with.  I don't think other implementations use the mapping metadata, and if they do, I don't know what procedure they'd have been following.  No matter how we define this, it's going to be different from someone's speculative interpretation.